nx.json

```diff
@@ -14,6 +14,10 @@
     "check:fix": {
       "dependsOn": ["^check:fix"],
       "cache": true
+    },
+    "test": {
+      "dependsOn": ["^build"],
+      "cache": true
     }
```

Comment:
This causes NX to build tested packages and their workspace dependencies before testing them.